### PR TITLE
Param/axis title menu layout

### PR DESCRIPTION
**Pull request by pk0t**
Added documentation for Axis Title menu for review and merging to main branch

### CODE REVIEW

File: docs/Axis-Title-menu-layout.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+=== "Display feature"
+
+    ![6a-hide-features](https://github.com/user-attachments/assets/e0d58454-09b0-4466-b51b-db4d98df0c65)
+
+=== "Hide feature"
+
+    ![6b-display-features](https://github.com/user-attachments/assets/a123b35a-f335-4cbb-832d-6b119734e6e2)
+
+## Selected features vs total features of a dataset
```
Comment (kennyAgricultureVic) [inline docs/Axis-Title-menu-layout.md:77]: Please change this to "Requested features vs total features of a dataset"

File: docs/Axis-Title-menu-layout.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Axis Title menu layout
+Click on the axis title to open the Axis Title menu. This offers several functionalities, as shown below, enabling users to visualize data in a way that best aligns with their objectives.
+
+![Axis Title Menu Layout- final](https://github.com/user-attachments/assets/f4853b09-806a-451b-845b-d89acdfbc495)
```
Comment (kennyAgricultureVic) [inline docs/Axis-Title-menu-layout.md:4]: Update the image to show "Requested Features vs total features" to make things consistent

File: docs/Axis-Title-menu-layout.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    ![6a-hide-features](https://github.com/user-attachments/assets/e0d58454-09b0-4466-b51b-db4d98df0c65)
+
+=== "Hide feature"
+
+    ![6b-display-features](https://github.com/user-attachments/assets/a123b35a-f335-4cbb-832d-6b119734e6e2)
+
+## Selected features vs total features of a dataset
+
+When zooming in on a specific genomic segment, the Axis Title menu displays the number of features within the selected region relative to the total features on the chromosome.
```
Comment (kennyAgricultureVic) [inline docs/Axis-Title-menu-layout.md:79]: Lets replace this with something along the lines of this but also inside a note.
See here for more info
https://squidfunk.github.io/mkdocs-material/reference/admonitions/
`This is an advanced feature used to debug technical issues with pretzel. This is technical counter which keeps track of the number of features requested from the backend`

File: docs/Axis-Title-menu-layout.md (hunk 2 above)
Comment (pk0t) [reply docs/Axis-Title-menu-layout.md:4]: Updated gif

File: docs/Axis-Title-menu-layout.md (hunk 1 above)
Comment (pk0t) [reply docs/Axis-Title-menu-layout.md:77]: Updated

File: docs/Axis-Title-menu-layout.md (hunk 3 above)
Comment (pk0t) [reply docs/Axis-Title-menu-layout.md:79]: Updated

**Pull request merged**